brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-1064 Fix: Info Box - Need margin instead of padding.

### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-1064
- It is an issue in our next release. As per master, it should be a margin option. In this PR I have updated the margin wherever we use padding for icon/image settings.

### Screenshots

- Editor - https://share.getcloudapp.com/xQuwLGmd
- Frontend - https://share.getcloudapp.com/E0uynDN6

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- At first, try the same padding option in the next release.
- Drag and drop the Info box, and add padding on the desktop from the Icon/Image Style tab.
- Check the editor and frontend. Adding 20 padding, icons not visible on both the editor and frontend.
- Now switch to above mention branch and refresh the editor and frontend after Regenerate Assets.
- You will see the icon is visible in both the editor and frontend.

### Checklist:
